ibu-imager/installation_configuration_files/scripts/installation-configuration.sh

```diff
@@ -25,6 +25,26 @@ fi
 echo "${RELOCATION_CONFIG_PATH} has been found"
 # Replace this with a function that loads values from yaml file
 set +o allexport
+CLUSTER_NAME=$(jq -r '.cluster_name' "${RELOCATION_CONFIG_PATH}"/clusterinfo/manifest.json)
+DOMAIN=$(jq -r '.domain' "${RELOCATION_CONFIG_PATH}"/clusterinfo/manifest.json)
+NEW_DOMAIN="${CLUSTER_NAME}.${DOMAIN}"
```

Comment:
The entire "original SNO" vs "seed SNO" vs "new sno" trinity is confusing enough, let's make sure we use very explicit names here to avoid confusion, e.g.:

```suggestion
NEW_CLUSTER_NAME=$(jq -r '.cluster_name' "${RELOCATION_CONFIG_PATH}"/clusterinfo/manifest.json)
NEW_CLUSTER_BASE_DOMAIN=$(jq -r '.domain' "${RELOCATION_CONFIG_PATH}"/clusterinfo/manifest.json)
NEW_CLUSTER_FULL_DOMAIN="${NEW_CLUSTER_NAME}.${NEW_CLUSTER_BASE_DOMAIN}"
```

Reply:
Also `SEED_DOMAIN` in other parts of this PR should be `SEED_FULL_DOMAIN` for consistency